packages/firestore/exp/src/api/database.ts

```diff
@@ -131,11 +143,10 @@ export class Firestore extends LiteFirestore
   }
 
   _clearPersistence(): Promise<void> {
-    if (this._firestoreClientPromise !== undefined && !this._terminated) {
+    if (this._deferredInitialization !== undefined && !this._terminated) {
       throw new FirestoreError(
         Code.FAILED_PRECONDITION,
-        'Persistence can only be cleared before the Firestore instance is ' +
-          'initialized or after it is terminated.'
+        'Persistence cannot be cleared after this Firestore instance is initialized.'
```

Comment:
This statement is not entirely correct, since you *can* clear persistence after terminating the client. I recommend using the original wording.

Reply:
I changed the error here to match the error message in the existing SDK. I reverted this change and changed the error message in the main SDK.